### DISCUSSION TITLE
Include reading of the response body in the request timeout

### DISCUSSION THREAD

**Discussion by GregoryAlbouy** [benchttp/runner#10] [Features decisions]
See [this thread](https://github.com/benchttp/runner/pull/8#discussion_r787961703) in #8.

**Comment by c1-ra** [comment]
I think we should include reading of the response body in the request timeout at first, because as you said, it is part of an endpoint's performance.
And if we have the time, after a first MVP, we can add a flag to let the user choose if they want to include it or not.
